stackstorm-nexus3/actions/lib/base.py

```diff
@@ -0,0 +1,178 @@
+import sys
+from exception import *
```

Comment:
Star imports are frowned upon. Can you explicitly list out everything you're importing?

Reply:
Same a few lines down.